Include/import.h

```diff
@@ -43,6 +43,11 @@ PyAPI_FUNC(PyObject *) PyImport_AddModuleObject(
 PyAPI_FUNC(PyObject *) PyImport_AddModule(
```

Comment:
Also mark `PyImport_AddModule` and `PyImport_AddModuleObject` as deprecated with the `Py_DEPRECATED(3.13)` macro?

Reply:
I changed my mind, I prefer to deprecate the function in a separated PR.